sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/PostLedgerEntryOperation.cs

```diff
@@ -88,5 +88,10 @@ async ValueTask<OperationState> IOperation.UpdateStateAsync(bool async, Cancella
 
         /// <inheritdoc />
         public override bool HasCompleted => _operationInternal.HasCompleted;
+
+        /// <summary>
+        /// To get the Id of the operation.
```

Comment:
```suggestion
        /// Gets the Id of the operation.
```

Reply:
Let's please update all instances of this comment to follow the common form.   It appears in our public docs and should be consistent with other types/members.